src/passes/pass.cpp

```diff
@@ -192,7 +193,7 @@ void PassRunner::run() {
       std::cerr << "final module does not validate\n";
       abort();
     }
-    passDebug = currPassDebug;
+    recursion--;
```

Comment:
Yeah, maybe the nested versions will want to parallelize sub-passes differently too.